Tests/LinuxMain.swift

```diff
@@ -1 +0,0 @@
-fatalError("Use `swift test --enable-test-discovery` to run tests")
```

Comment:
This file is no longer needed since test discovery is enabled by default in Swift 5.4 on non-Apple platforms.